app/server/src/main/scala/org/bitcoins/server/NodeRoutes.scala

```diff
@@ -4,16 +4,37 @@ import akka.actor.ActorSystem
 import akka.http.scaladsl.server._
 import akka.http.scaladsl.server.Directives._
 import akka.stream.ActorMaterializer
-import org.bitcoins.node.Node
+import org.bitcoins.node.{NeutrinoNode, Node}
+
+import scala.util.{Failure, Success}
 
 case class NodeRoutes(node: Node)(implicit system: ActorSystem)
     extends ServerRoute {
+  import system.dispatcher
   implicit val materializer = ActorMaterializer()
 
   def handleCommand: PartialFunction[ServerCommand, StandardRoute] = {
     case ServerCommand("getpeers", _) =>
       complete {
         Server.httpSuccess("TODO implement getpeers")
       }
+    case ServerCommand("rescan", arr) =>
```

Comment:
I know there isn't any unit tests for this stuff yet, but can you please begin to implement them?